### PR TITLE
reset.css add to repository

### PR DESCRIPTION
**Pull request by pczern**

### CODE REVIEW

**Comment by gtomitsuka** [comment]
Seems fine.
I think you should add more commits to the pull request.

File: server/reset.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
```
Comment (thomas-daniels) [inline server/reset.css:6]: This is a very long list of elements. You can use `*` instead of listing all elements here.

**Comment by gtomitsuka** [comment]
Yes, that would save space. I'll add a static files folder for CSSes, later. So we can separate the main HTML and JS from the css.

File: server/reset.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
```
Comment (Unihedro) [inline server/reset.css:31]: Is there a particular reason for having this element here? This does no influence on the user agent, plus default values of `line-height` for desktop browsers are different for a decent reason, for example desktop browsers (including FireFox) uses default value of roughly 1.2 for standard display. Unless you're using a monospaced font, hardcoding this value won't do anything.

**Comment by gtomitsuka** [comment]
There is a new pull request with the changes

**Pull request closed without merging**